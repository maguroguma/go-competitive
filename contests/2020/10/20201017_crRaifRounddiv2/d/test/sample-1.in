6
2 0 3 0 1 1
