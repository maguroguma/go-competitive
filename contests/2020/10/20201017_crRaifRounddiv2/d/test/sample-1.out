5
2 1
2 5
3 3
3 6
5 6
